### ISSUE TITLE
cometchat/arrowchat Integration

### ISSUE THREAD

**Issue by przemo5050** [bootstrapguru/socialite-requests#21]
Make  integration with cometchat/arrowchat 
In this good script social network will be great to show chat rooms with all users

**Comment by Nivagus** [comment]
Yeah I'm Looking too script with this integration on codecanyon :_;

**Comment by Bojilovx** [comment]
cometchat is better than arrow 👍
